src/components/Menu/index.js

```diff
@@ -42,64 +58,87 @@ const Menu = ({ location }) => {
-                />
-                <MenuDropdownLink
-                  link="exploration"
-                  activeSubMenu={pathname === '/exploration'}
-                />
-                <MenuDropdownLink
-                  link="texts"
-                  activeSubMenu={pathname === '/texts'}
-                />
-              </React.Fragment>
-            ) : null}
+            <React.Fragment>
+              <MenuDropdownLink
+                link="about"
+                activeSubMenu={pathname === '/about'}
+              />
+              <MenuDropdownLink
+                link="exploration"
+                activeSubMenu={pathname === '/exploration'}
+              />
+              <MenuDropdownLink
+                link="texts"
+                activeSubMenu={pathname === '/texts'}
+              />
+            </React.Fragment>
+          ) : null}
         </ul>
+
         {/* ############# CONTACT SECTION ############# */}
         <ul className={css(styles.subMenuCol)}>
           <li
             onClick={() => navigate('/find-us')}
             className={css(styles.subMenuItem)}
           >
-            <span className={css(styles.subMenuLink)}>contact</span>
+            <span
+              className={
+                pathname === '/find-us' ||
+                pathname === '/press' ||
+                pathname === '/retailers'
+                  ? css(styles.subMenuLinkBlack)
+                  : css(styles.subMenuLink)
+              }
+            >
+              contact
+            </span>
           </li>
           {pathname === '/find-us' ||
           pathname === '/press' ||
           pathname === '/retailers' ? (
-              <React.Fragment>
-                <MenuDropdownLink
-                  alias="find us"
-                  link="find-us"
-                  activeSubMenu={pathname === '/find-us'}
-                />
-                <MenuDropdownLink
-                  link="press"
-                  activeSubMenu={pathname === '/press'}
-                />
-                <MenuDropdownLink
-                  link="retailers"
-                  activeSubMenu={pathname === '/retailers'}
-                />
-              </React.Fragment>
-            ) : null}
+            <React.Fragment>
+              <MenuDropdownLink
+                alias="find us"
+                link="find-us"
+                activeSubMenu={pathname === '/find-us'}
+              />
+              <MenuDropdownLink
+                link="press"
+                activeSubMenu={pathname === '/press'}
+              />
+              <MenuDropdownLink
+                link="retailers"
+                activeSubMenu={pathname === '/retailers'}
+              />
+            </React.Fragment>
+          ) : null}
         </ul>
-      </Nav>
-      <Nav pullRight>
-        <ul className={css(styles.subMenuCol)}>
+      </div>
+
+      <div className={css(styles.shopWrap)}>
+        <ul className={css(styles.subMenuCol2)}>
```

Comment:
@Alphascythian shop does not have a submenu, why does it need to have a class called `subMenuCol2`?